cinema/tests/test_movie_api.py

```diff
@@ -157,3 +158,164 @@
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
+
+
+class UnauthenticatedMovieAoiTest(TestCase):
```

Comment:
The class name `UnauthenticatedMovieAoiTest` seems to have a typo. It should likely be `UnauthenticatedMovieApiTest`.